### PR TITLE
completed as requested

### PR DESCRIPTION
**Pull request by guncebektas**

### CODE REVIEW

**Comment by guncebektas** [comment]
@filipenevola a kindly reminder. If you want to update anything, I will be glad to help

File: README.md [hunk 1]
```diff
@@ -10,6 +10,14 @@
 - Last Updated At: Apr/07/2021
 - Meteor Version: 2.1.1
 
+#### Complex Todos
```
Comment (StorytellerCZ) [inline README.md:13]: Should probably be `Complex Svelte Todos`

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -10,6 +10,14 @@
...
+- Demo: [complex-todos-svelte.meteorapp.com/](https://complex-todos-svelte.meteorapp.com)
+- Repository: [guncebektas/complex-todos-svelte](https://github.com/guncebektas/complex-todos-svelte)
+- Why: To show how to use Meteor in real world problems.
+- Stack: Meteor, Svelte, Bootstrap, MongoDB, Cypress
+- Last Updated At: May/15/2021
+- Meteor Version: 2.1.1
```
Comment (StorytellerCZ) [inline README.md:19]: Is the date correct?
Also while at it, maybe it would be great to update to the latest Meteor or at least Meteor 2.2.3

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (filipenevola) [reply README.md:13]: Yes, it would be better.

**Review by filipenevola — APPROVED** [review]
I'm not a Svelte developer so I can't analyze the details but the demo app is working as expected :)

File: README.md (hunk 2 above)
Comment (guncebektas) [reply README.md:19]: I will update the meteor and the date too

**Comment by guncebektas** [comment]
hi, @StorytellerCZ I have completed your request and found a bug on UI. All are fixed.  
Thanks for the review.

**Review by filipenevola — APPROVED** [review]

**Pull request merged**